crates/sui-pg-db/src/lib.rs

```diff
@@ -65,15 +78,15 @@ impl Db {
 
     /// Retrieves a connection from the pool. Can fail with a timeout if a connection cannot be
     /// established before the [DbArgs::connection_timeout] has elapsed.
-    pub async fn connect(&self) -> anyhow::Result<Connection<'_>> {
+    pub async fn connection(&self) -> anyhow::Result<Connection<'_>> {
```

Comment:
This should go away now, right? Because you can use `Store::connect` everywhere?